### PR TITLE
add sentry

### PR DESCRIPTION
**Pull request by atytko**
added and configured Sentry

### CODE REVIEW

File: config/initializers/sentry.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+Sentry.init do |config|
+  config.dsn = 'https://e8bd9a43b666488db6bee4975626669d@o949860.ingest.sentry.io/5898547'
```
Comment (kodowicz) [inline config/initializers/sentry.rb:4]: configs with secret keys should not be public :) Try to play with [ENV](https://www.rubyguides.com/2019/01/ruby-environment-variables/)

Comment (agusiol) [inline config/initializers/sentry.rb:4]: You shouldn't push data like this to your GitHub repository. You can use for example dotenv to mange environmental variables :).

Comment (atytko) [reply config/initializers/sentry.rb]: thanks, I added dotenv gem and made the key private :)

Comment (atytko) [reply config/initializers/sentry.rb]: thanks, I added dotenv gem :)

**Review by agusiol — APPROVED** [review]

**Review by kodowicz — APPROVED** [review]
great job!

**Comment by wmaciejak** [comment]
Really good job @atytko 👏

**Pull request merged**